README.md

```diff
@@ -25,23 +27,26 @@ module "account_setup" {
 
 ## Inputs
 
-| Name | Description | Type | Default | Required |
-|------|-------------|:----:|:-----:|:-----:|
-| allow_users_to_change_password | Whether to allow users to change their own password | string | `true` | no |
-| enable_account_password_policy | Enable custom (strict) password policy. | string | `true` | no |
-| enable_admin_group | Create an admin group. | string | `true` | no |
-| enable_mfa | Enable to force MFA usages. | string | `true` | no |
-| hard_expiry | Whether users are prevented from setting a new password after their password has expired (i.e. require administrator reset) | string | `false` | no |
-| max_password_age | The number of days that an user password is valid. | string | `33` | no |
-| minimum_password_length | Minimum length to require for user passwords. | string | `32` | no |
-| password_reuse_prevention | The number of previous passwords that users are prevented from reusing. | string | `true` | no |
-| require_lowercase_characters | Whether to require lowercase characters for user passwords. | string | `true` | no |
-| require_numbers | Whether to require numbers for user passwords. | string | `true` | no |
-| require_symbols | Whether to require symbols for user passwords. | string | `true` | no |
-| require_uppercase_characters | Whether to require uppercase characters for user passwords. | string | `true` | no |
+| Name                           | Description                                                                                                                 | Type   | Default     | Required |
+| ------------------------------ | --------------------------------------------------------------------------------------------------------------------------- | :----: | :---------: | :------: |
+| admin_group_name               | Name of the admin group..                                                                                                   | string | `admins`    | no       |
+| allow_users_to_change_password | Whether to allow users to change their own password                                                                         | string | `true`      | no       |
+| enable_account_password_policy | Enable custom (strict) password policy.                                                                                     | string | `true`      | no       |
+| enable_admin_group             | Create an admin group.                                                                                                      | string | `true`      | no       |
+| enable_mfa                     | Enable to force MFA usages.                                                                                                 | string | `true`      | no       |
+| enable_read_only               | Creates a group with read-only IAM policy assigned to it                                                                    | string | `true`      | no       |
```

Comment:
Description can be confusing, can you make it clear that is is about a global ReadOnly predefined AWS policy